aws-samples/amazon-chime-sdk
# Bump up the version of chime sdks in react meeting demo

**Issue #:**
Need to update the version of chime sdks for meeting demo to latest .

**Description of changes:**
Update the version of chime sdks for meeting demo.
* Update `amazon-chime-sdk-component-library-react` from 2.8.0 to 2.9.0
* Update `amazon-chime-sdk-js` from 2.13.0 to 2.16.1

**Testing**

1. How did you test these changes?
* Run the meeting demo and make an E2E test. The demo works well. 
* Check the VoiceFocus function and it works well.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
N/A

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
